cpp/libcudf_kafka/include/cudf_kafka/kafka_consumer.hpp

```diff
@@ -149,9 +149,11 @@ class kafka_consumer : public cudf::io::datasource {
   /**
    * @brief Query the Kafka broker for the assigned topic Metadata
    *
+   * @param[in] Retrieves metadata for only a specific topic.
```

Comment:
@param requires a parameter name.  Also, maybe the name of this parameter should just be `topic_name`?

```suggestion
   * @param[in] specific_topic The name of the topic for which to retrieve metadata
```